src/frontend/XAPI/index.ts

```diff
@@ -0,0 +1,94 @@
+// https://liveaspankaj.gitbooks.io/xapi-video-profile/content/statement_data_model.html
```

Comment:
There should probably be a quick summary of why we need this link and what it contains